mlflow/store/tracking/abstract_store.py

```diff
@@ -1,11 +1,14 @@
 from abc import ABCMeta, abstractmethod
+from concurrent.futures import ThreadPoolExecutor
 from typing import List, Optional
 
 from mlflow.entities import DatasetInput, ViewType
 from mlflow.store.entities.paged_list import PagedList
 from mlflow.store.tracking import SEARCH_MAX_RESULTS_DEFAULT
 from mlflow.utils.annotations import developer_stable, experimental
 
+_ASYNC_DATA_LOGGING_THREAD_POOL = ThreadPoolExecutor(max_workers=2)
```

Comment:
can we add some inline comments about how this thread pool works and is used?

Reply:
I have added basic description, not sure thats what you are looking for. Let me know.